### PR TITLE
Add Orientations to GRiTS

### PR DESCRIPTION
**Pull request by NoahMontrose**
This PR finds orientations and populates orientation attributes in the gsd file.

### CODE REVIEW

File: grits/coarsegrain.py [hunk 1]
```diff
@@ -640,14 +708,26 @@ def save(self, cg_gsdfile, start=0, stop=None):
...
                 for i, inds in enumerate(self.mapping.values()):
                     position += [np.mean(unwrap_pos[x], axis=0) for x in inds]
                     mass += [sum(s.particles.mass[x]) for x in inds]
-
+                    if self.aniso_beads:
+                        for x in inds:
+                            masses = s.particles.mass[x]
+                            hmass = element_from_symbol('H').mass
+                            positions = s.particles.position[x]
+                            heavy_positions = positions[np.where(masses > hmass)]
```
Comment (chrisjonesBSU) [inline grits/coarsegrain.py:724]: This won't work if we're using reduced units, where the masses will all be <= 1

File: grits/coarsegrain.py [hunk 2]
```diff
@@ -640,14 +708,26 @@ def save(self, cg_gsdfile, start=0, stop=None):
...
                 )
                 for i, inds in enumerate(self.mapping.values()):
                     position += [np.mean(unwrap_pos[x], axis=0) for x in inds]
                     mass += [sum(s.particles.mass[x]) for x in inds]
-
+                    if self.aniso_beads:
+                        for x in inds:
+                            masses = s.particles.mass[x]
+                            hmass = element_from_symbol('H').mass
+                            positions = s.particles.position[x]
```
Comment (chrisjonesBSU) [inline grits/coarsegrain.py:723]: Do we need to use the unwrapped positions here? They are defined a few lines up

File: grits/coarsegrain.py [hunk 3]
```diff
@@ -640,14 +708,26 @@ def save(self, cg_gsdfile, start=0, stop=None):
...
+                            heavy_positions = positions[np.where(masses > hmass)]
+                            heavy_positions = f_box.wrap(heavy_positions)
```
Comment (chrisjonesBSU) [inline grits/coarsegrain.py:725]: `s.particles.position` already is the wrapped positions if I recall correctly.

**Review by chrisjonesBSU — COMMENTED** [review]
Left a couple comments about issues arising for units. I'm also confused about the position arrays. Do we want to use unwrapped positions for calculating things like the orientation? Is there a reason we don't want to include hydrogen atom positions when calculating orientation?

**Comment by RainierBarrett** [comment]
Closing. Addressed by #54.

**Pull request closed without merging**